webapp/src/test/java/com/box/l10n/mojito/service/tm/TMServiceTest.java

```diff
@@ -1409,8 +1409,8 @@ public void testExportAssetAsXLIFF() throws RepositoryNameAlreadyUsedException {
 
   private String removeIdsAndDatesFromJson(String xliff) {
     String cleanXliff = xliff.replaceAll("\"id\":\\d+,?", "");
-    cleanXliff = cleanXliff.replaceAll("\"createdDate\":\\d+,?", "");
-    cleanXliff = cleanXliff.replaceAll("\"lastModifiedDate\":\\d+,?", "");
+    cleanXliff = cleanXliff.replaceAll("\"createdDate\":\\d+\\.\\d+,?", "");
```

Comment:
are XLIFF outputs changed? This could be silencing the difference because we do exact match comparison?